### DISCUSSION TITLE
Google Voice - Tips and Tricks

### DISCUSSION THREAD

**Discussion by shaunwbell** [NOAA-PMEL/WRC-TUG#12] [General]
As NOAA/WRC/PMEL rolls out Google Voice - Come back here to share tips and ask questions (or ping the Google Spaces TUG chatroom).

- you only need to have your gmail tab open on a supported browser (Chrome, Firefox, Opera, Edge?) to receive calls.  It does not appear you need the google voice page open (although there are more features on that page than in the gmail sidebar)
- take advantage of "Working Hours" in Google Calendar - then in Google Voice, turn on the option to use these to defer calls to voicemail outside of working hours.  Same can be set so that all calls get sent to voicemail when your calendar events are set to "out of office"
- take advantage of "Do Not Disturb" in the Google Voice Settings to send all calls to voicemail.  This is not the same as the  in the top bar of the Gmail page to the right of the search bar - that is for silencing Google Chat notifications.

Others are welcome to contribute as they test or use Google Voice

**Comment by shaunwbell** [comment]
It has been commented by a few that the settings to **Do Not Disturb** settings are not working as one would expect.

From the [Google Voice Settings Page](https://voice.google.com/u/1/settings) There are two primary settings to modify here, the first is an overarching "Do Not Disturb" slider.  All calls will be sent to voicemail regardless of other settings if this is toggled on. The other two are Calendar based and will send messages to your voicemail based on your *Working Hours* and *Out of Office* settings - which are specified by your Calendar.

I personally have had success here, so welcome any further discussion should anyone else be experiencing difficulties.

**Comment by adi-hanein** [comment]
I attended the NOAA Google Voice Training today (1/19) and here's some tips I learned:

- Did you know you can pin a tab in Chrome? I didn't! The trainer recommended pinning the voice.google.com tab
- You can mark a phone call or text message as spam by clicking the three dots on the upper right hand side. It won't block it automatically but will send to a list of suspected spam numbers. 
- The trainer recommended not turning on Do Not Disturb and just using the Google Calendar integration for specific times to send straight to voicemail. 
- If you're on a call and get another call, it'll send the call directly to voicemail (similar to do not disturb). Had that already happen which was a little annoying since it just shows up as a red icon of a missed call. May see a change in Q2? Unclear. 
- Outgoing phone calls on your cell: There is a setting in Google Voice (make and receive telephone calls; prefer wifi/mobile data)..he showed the app though....
- International calls are by default turned off (NOAA has not enabled this). Canada is fine apparently. Talk to CNSD if you need this enabled. 
- If you're curious, this is paid for by a direct bill to the line offices. 

You can probably skip the first 20 minutes of future ones as he walked through placing a phone call first, how to check your microphone, etc...before getting to settings and then doing Q&A! **Pay attention at the beginning for the Q&A form** because there is no chat feature. The NOAA person basically said go to your local IT for questions in the future.

**Reply by shaunwbell** [reply]
Thanks @adi-hanein - some good tidbits hidden in that unfortunately long session.
